src/Core/src/Core/IProgress.cs

```diff
@@ -11,5 +11,10 @@ public interface IProgress : IView
 		/// Progress values less than 0 will be clamped to 0, values greater than 1 will be clamped to 1.
 		/// </summary>
 		double Progress { get; }
+
+		/// <summary>
+		/// Get the color of the progress bar.
+		/// </summary>
+		Color ProgressColor { get; }
```

Comment:
This should be a Brush.

Reply:
Totally agree. However, can we make it progressive? Start by merging this and other PRs related to Color, in order to complete Handlers and allow in Previews to work and port code in a simple way, and as we have to do with other important properties such as  convert TextColor to Foreground, update little by little .

Reply:
Yeah i think we could, i added a issue for this https://github.com/dotnet/maui/issues/2403